avm/ptn/azd/container-apps/tests/e2e/defaults/main.test.bicep

```
@@ -0,0 +1,75 @@
+targetScope = 'subscription'
+
+metadata name = 'With zoneRedundant enabled'
+metadata description = 'This instance deploys the module with zoneRedundant enabled.'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-container-apps-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'casmin'
```

```suggestion
param serviceShort string = 'acazr'
```

Note: Requires a regeneration of the docs